package.json

```diff
@@ -68,12 +68,14 @@
     "once": "^1.3.1",
     "pumpify": "^1.3.3",
     "safe-buffer": "^5.1.1",
+    "snakeize": "^0.1.0",
     "stream-events": "^1.0.1",
     "string-format-obj": "^1.0.0",
     "through2": "^2.0.0"
   },
   "devDependencies": {
     "@google-cloud/nodejs-repo-tools": "^2.0.7",
+    "@google-cloud/pubsub": "*",
```

Comment:
Is depending on `*` what we want?

Reply:
Actually I had a specific version tied to this, but @stephenplusplus requested I use `*` instead.